tests/test_enterprise/api/test_views.py

```diff
@@ -431,7 +431,7 @@ def test_post_enterprise_customer_user_logged_out(self):
 
     @ddt.data(
         {'is_staff': True, 'user_exists': True, 'ecu_exists': True, 'pending_ecu_exists': False, 'status_code': 204},
-        {'is_staff': True, 'user_exists': True, 'ecu_exists': False, 'pending_ecu_exists': True, 'status_code': 204},
+        {'is_staff': True, 'user_exists': True, 'ecu_exists': False, 'pending_ecu_exists': True, 'status_code': 201},
```

Comment:
As the view is implemented, if we post to the pending enterprise customer user API for a user that already exists and has a PendingEnterpriseCustomerUser record, we opt to link the user by creating an `EnterpriseCustomerUser` record for that user/customer.

As such, we would expect a 201 in this case, not a 204.